website/source/api/system/auth.html.md

```diff
@@ -194,11 +194,11 @@ can be achieved without `sudo` via `sys/mounts/auth/[auth-path]/tune`._
 - `description` `(string: "")` – Specifies the description of the mount. This
   overrides the current stored value, if any.
 
-- `audit_non_hmac_request_keys` `(string:"")` - Specifies the comma-separated
+- `audit_non_hmac_request_keys` `(array: [])` - Specifies the comma-separated
```

Comment:
[Optional] We could just say that its a list/array of keys and mention that comma-separated string is also accepted. This might reduce confusion.